src/box/lua/schema.lua

```diff
@@ -330,22 +342,61 @@ local function feedback_save_event(event)
     end
 end
 
+-- Extract isolation names from C, create and return name->ID map.
+local function collect_txn_isolation()
+    local res = {}
+    for i = 0, builtin.txn_isolation_level_MAX - 1 do
+        res[ffi.string(builtin.txn_isolation_level_strs[i])] = i
+    end
+    return res;
+end
+
+-- Convert to numeric the value of tnx isolation, raise an error if failed.
+local function normalize_txn_isolation(txn_isolation)
+    if type(txn_isolation) == 'string' then
+        txn_isolation = box.txn_isolation[txn_isolation]
+        if not txn_isolation then
+            box.error(box.error.ILLEGAL_PARAMS,
+                      "txn_isolation must be one of box.txn_isolation" ..
+                      " (keys or values)")
```

Comment:
I think it would be better to use the same pattern here: https://github.com/tarantool/tarantool/blob/f1d750f036956873c79bf5de51f77b38da25ae0f/src/lua/log.lua#L252-L255